testutils/mocks/Logger.go

```diff
@@ -0,0 +1,81 @@
+package mocks
```

Comment:
Also, how about just a root-level mocks folder instead of `testutils/mocks`? I think mocks imply that there only testing utilities.